core/src/main/java/io/grpc/StreamTracer.java

```diff
@@ -62,32 +62,45 @@ public void inboundMessage() {
   }
 
   /**
+   * An outbound message has been completely serialized, revealing its wire size.
+   */
+  public void outboundMessageSerialized(long wireSize) {
```

Comment:
@bogdandrutu for now we could provide accurate wire size and timing at the same time. But soon with stream-wide compression we will be left with two options:
 1. Provide accurate timing only. NetworkEvent could be added for each message, but the actual wire size is unknown. We could know the uncompressed size for outbound, but will have difficulty (but not impossible) in knowing the uncompressed size for inbound.
 2. Provide accurate wire size and timing for each HTTP2 frame.